projects/observability/src/shared/graphql/model/schema/filter/global-graphql-filter.service.ts

```diff
@@ -5,12 +5,27 @@ import { GraphQlFilter } from './graphql-filter';
 export class GlobalGraphQlFilterService {
   private readonly filterRules: Map<symbol, FilterRule> = new Map();
 
-  public getGlobalFilters(scope: string): GraphQlFilter[] {
-    return Array.from(this.filterRules.values()).flatMap(rule => rule.filtersForScope(scope));
+  public getGlobalFilters(scope: string, filtersToExclude: symbol[] = []): GraphQlFilter[] {
```

Comment:
This feels like we're digging the hole even deeper - the idea of global filters is they're opaque - we shouldn't be requesting to exclude them.